newsletter-june-2023.rst

```diff
@@ -10,8 +10,14 @@ Read the Docs newsletter - June 2023
 News and updates
```

Comment:
Maybe is worth mentioning that we have migrated all custom domains to have 1.2 as the minimum TLS version?

The `/en/`to `/en/latest/` redirect could also be interesting to mention